interfaces/cython/cantera/test/test_reactor.py

```diff
@@ -1542,3 +1542,39 @@ def test_ConstPressureReactor(self):
         self.assertEqual(states.X[-2], 1)
         for i in range(3,7):
             self.assertNear(states.T[i], states.T[2])
+
+
+class ImplicitSurfChemTest(utilities.CanteraTest):
+    def test_advance_coverages_parameters(self):
+        # create gas and interface
+        gas1 = ct.Solution('ptcombust.xml', 'gas')
+        surf = ct.Interface('ptcombust.xml', 'Pt_surf', [gas1])
+
+        # first, test max step size & max steps
+        dt = 1.0
+        max_steps = 10
+        max_step_size = dt / (max_steps + 1)
+        # this should throw an error, as we can't reach dt
+        try:
+            surf.advance_coverages(
+                dt=dt, max_step_size=max_step_size, max_steps=max_steps)
+            self.fail()
+        except ct.CanteraError:
+            pass
+
+        # next, run with different tolerances
+        gas1 = ct.Solution('ptcombust.xml', 'gas')
+        surf = ct.Interface('ptcombust.xml', 'Pt_surf', [gas1])
```

Comment:
This should probably be a separate test method, with the initialization of the `gas1` and `surf1` objects done in the `setUp` method.